python-package/lightgbm/engine.py

```diff
@@ -370,6 +370,8 @@ def cv(params, train_set, num_boost_round=10,
 
     if metrics:
         params.setdefault('metric', [])
+        if isinstance(params['metric'], string_type):
```

Comment:
I don't think user will set both params['metric'] and metrics intensionally, and expect both of them work. I guess most of time, user copied the params he used before and didn't notice 'metric' is in it, or can't figure out which to use, then set them with same thing and get duplicate outputs. So I think override and specify it in docs is enough.